### PR TITLE
[ANCHOR-223] Update git hook

### PR DESCRIPTION
**Pull request by Ifropc**
<details>
  <summary>PR Checklist</summary>

### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `paymentservice.stellar`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
</details>

### What

Update git hook

### Why

You can't commit not formatted code anymore

### Known limitations

[TODO or N/A]

### CODE REVIEW

File: scripts/pre-commit.sh [hunk 1]
```diff
@@ -1,15 +1,7 @@
 #!/bin/sh
 
-# find staged files
-stagedFiles=$(git diff --staged --name-only)
+R='\033[0;31m'
```
Comment (lijamie98) [inline scripts/pre-commit.sh:3]: This looks black magic to me. :-)

**Review by lijamie98 — COMMENTED** [review]
LGTM.

File: scripts/pre-commit.sh (hunk 1 above)
Comment (Ifropc) [reply scripts/pre-commit.sh:3]: It's just red color so it's a bit more pretty

Comment (lijamie98) [reply scripts/pre-commit.sh:3]: It looks like I forgot to approve.

**Review by lijamie98 — APPROVED** [review]

**Pull request merged**